mvvmfx/src/main/java/de/saxsys/mvvmfx/internal/viewloader/ViewLoaderReflectionUtils.java

```diff
@@ -386,6 +380,11 @@ static Object injectScopeIntoField(Field scopeField, Object viewModel, ContextIm
         }
         try {
             final Method initMethod = viewModel.getClass().getMethod("initialize");
+            // if there is a @PostConstruct annotation, throw an exception to prevent double injection
+            if(initMethod.isAnnotationPresent(PostConstruct.class)) {
+                throw new IllegalStateException(String.format("double injection on initialize method of ViewModel [%s]. " +
```

Comment:
Maybe we can improve the error message. A suggestion:
"initialize method of ViewModel [%s] is annotated with @PostConstruct. This will lead to unexpected behaviour and duplicate initialization. Please rename the method or remove the @PostConstruct annotation. See mvvmFX wiki for more details: https://github.com/sialcasa/mvvmFX/wiki/Dependency-Injection#livecycle-postconstruct"